test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -377,6 +371,78 @@ internal enum FILE_CREATE_FLAGS
         this.AssertNoDiagnostics();
     }
 
+    // Slow span can't safely create a span from a ref when the struct is on the heap.
+    [Fact]
+    public void CodeGenerationForFixedLengthInlineArrayWithSlowSpan()
+    {
+        const string expected = @"
+    internal partial struct MainAVIHeader
+    {
+        internal uint dwMicroSecPerFrame;
+        internal uint dwMaxBytesPerSec;
+        internal uint dwPaddingGranularity;
+        internal uint dwFlags;
+        internal uint dwTotalFrames;
+        internal uint dwInitialFrames;
+        internal uint dwStreams;
+        internal uint dwSuggestedBufferSize;
+        internal uint dwWidth;
+        internal uint dwHeight;
+        internal __dwReserved_4 dwReserved;
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
```

Comment:
@tannergooding were you saying we don't need to specify the pack size?

Reply:
Right. Inline arrays are just like regular arrays and the natural padding of a struct is correct.

Reply:
(assuming the metadata doesn't specify otherwise)

Reply:
Saw that too. I can add this change to this PR, no problem. :+1: